data/common.yaml

```diff
@@ -3,4 +3,4 @@ lookup_options:
   puppet_data_service::server::pds_token:
     convert_to: "Sensitive"
 
-puppet_data_service::server::package_source: 'https://github.com/puppetlabs/puppet-data-service/releases/download/v0.1.0/pds-server-0.1.0-1.x86_64.pe-2021.4.0.rpm'
+puppet_data_service::server::package_source: 'https://github.com/puppetlabs/puppet-data-service/releases/download/v0.1.1/pds-server-0.1.1-1.pe.2021.5.0.el7.x86_64.rpm'
```

Comment:
@timidri  Just wondering if we should use the PDS RPM of the "lowest" version available as the default source?

Reply:
@mcka1n @mcka1n which one should we pick? https://github.com/puppetlabs/puppet-data-service/releases ?

Reply:
What about:
```suggestion
puppet_data_service::server::package_source: 'https://github.com/puppetlabs/puppet-data-service/releases/download/v0.1.1/pds-server-0.1.1-1.pe.2019.8.10.el7.x86_64.rpm'
```

[pds-server-0.1.1-1.pe.2019.8.10.el7.x86_64.rpm](https://github.com/puppetlabs/puppet-data-service/releases/download/v0.1.1/pds-server-0.1.1-1.pe.2019.8.10.el7.x86_64.rpm)

Reply:
@mcka1n So will this version work on the latest PE? I'm confused...